.gitignore

```diff
@@ -31,6 +31,11 @@
 /backend/log
 /backend/tmp
 
+# Ignore v3 api docs
+/backend/.yardoc
```

Comment:
we definitely don't want to checking the generated files, but do we want to ignore the config files?

Reply:
I think we can safely ignore them (and I think we should).

Before I submitted the PR I took out the folder out and reran "yardoc --api api.v3 --output-dir docs/api/v3" and it successfully recreated itself.

I also believe the contents might change on different runs (for example, there is a checksums file), but I did not verify that one way or the other.

Happy to add it back in, though.  Just let me know.